core/controllers/base.py

```diff
@@ -275,6 +275,10 @@ def render_json(self, values):
         json_output = json.dumps(values, cls=utils.JSONEncoderForHTML)
         self.response.write('%s%s' % (feconf.XSSI_PREFIX, json_output))
 
+    def get_logout_url(self, redirect_url_on_logout):
+        """Returns logout url."""
```

Comment:
Docstring should include Args and Returns sections; follow the format of existing docstrings in files in this codebase.

Reply:
done.